### PR TITLE
The implementation of dice loss

### PR DESCRIPTION
**Pull request by zjhellofss**
rt.

### CODE REVIEW

File: paddleseg/models/losses/dice_loss.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+from paddle import nn
+from paddleseg.cvlibs import manager
+import paddle.nn.functional as F
```
Comment (wuyefeilin) [inline paddleseg/models/losses/dice_loss.py:14]: 整体没问题，只要这边import的格式稍微改一下就行了， 具体可以参考cross_entroy_loss的格式

**Review by wuyefeilin — COMMENTED** [review]
It is good. Only a few format need to be fixed.

**Review by wuyefeilin — APPROVED** [review]
LGTM
